fe/fe-core/src/main/java/org/apache/doris/nereids/operators/plans/logical/LogicalSort.java

```diff
@@ -0,0 +1,103 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.nereids.operators.plans.logical;
+
+import org.apache.doris.nereids.exceptions.UnboundException;
+import org.apache.doris.nereids.operators.OperatorType;
+import org.apache.doris.nereids.trees.expressions.NamedExpression;
+import org.apache.doris.nereids.trees.expressions.Slot;
+import org.apache.doris.nereids.trees.plans.Plan;
+
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+    Logical Sort plan operator.
+ */
+public class LogicalSort extends LogicalUnaryOperator {
+
+    private List<SortItems> sortItems;
+
+    /**
+     * Constructor for SortItems.
+     */
+    public LogicalSort(List<SortItems> sortItems) {
+        super(OperatorType.LOGICAL_SORT);
+        this.sortItems = Objects.requireNonNull(sortItems, "sorItems can not be null");
+    }
+
+    @Override
+    public String toString() {
+        return "Sort (" + StringUtils.join(sortItems, ", ") + ")";
+    }
+
+    @Override
+    public List<Slot> computeOutput(Plan input) {
+        return sortItems.stream()
+                .map(namedExpr -> {
+                    try {
+                        return namedExpr.getSort().toSlot();
+                    } catch (UnboundException e) {
+                        throw new IllegalStateException(e);
+                    }
+                })
+                .collect(ImmutableList.toImmutableList());
```

Comment:
sort node's output should just return child's output, but not use sort item

Reply:
done